### ISSUE TITLE
Java 11 - exception while calling MonocleRobot->getScreenCapture

### ISSUE THREAD

**Issue by cristiannandrean** [TestFX/Monocle#83]
jdk: OpenJDK11U-jdk_x64_windows_hotspot_11.0.10_9
javafx version: 11.0.2
monocle version: jdk-11+26

JVM parameters:
-Dtestfx.robot=glass
-Dglass.platform=Monocle
-Dmonocle.platform=Headless
-Dprism.order=sw
-Dprism.text=t2k

code:  
```
  public static WritableImage takeScreenshot(final GlassRobot robot) {
        final Rectangle2D bounds = Screen.getPrimary().getVisualBounds();
        final int width = (int) bounds.getWidth();
        final int height = (int) bounds.getHeight();
        final int x = (int) bounds.getMinX();
        final int y = (int) bounds.getMinY();

        return robot.getScreenCapture(null, x, y, width, height, false);
    }
```
	
Expected result: screenhsot captured
Actual result: java.lang.UnsupportedOperationException
	at java.base/java.nio.IntBuffer.array(IntBuffer.java:1049)
	at com.sun.glass.ui.monocle.MonocleRobot.getScreenCapture(MonocleRobot.java:209)
	at com.sun.glass.ui.GlassRobot.getScreenCapture(GlassRobot.java:200)

**Comment by longtimeago** [comment]
the same for java 17

**Comment by shankalel** [comment]
Any fix or workaround would be highly appreciated for this issue.

**Comment by mvsoder** [comment]
We recently posted new Monocle builds for JFX 11 and 17: https://central.sonatype.com/artifact/org.testfx/openjfx-monocle/versions

**Comment by shankalel** [comment]
This issue is still not fixed with latest Monocle builds for JFX 11 and 17 postewd on 2024-02-11.
Looking forward eargely for this fix as it is blocking our Test Automation

**Comment by mvsoder** [comment]
Please note that this is not the 'official' Monocle repository. This repository is only for generating builds to support TestFX. To report bugs for Monocle, please visit, https://github.com/openjdk/jfx, and read the section on Issue Tracking.

**Comment by SaiPradeepDandem** [comment]
Any update regarding this issue?
This is still not fixed in Monocle [21.0.2](https://central.sonatype.com/artifact/org.testfx/openjfx-monocle/21.0.2) version :(
